python/qpid_dispatch/management/qdrouter.json

```diff
@@ -841,6 +841,46 @@
                     "type": "string",
                     "description": "Where to send log messages. Can be 'stderr', 'stdout', 'syslog' or a file name.",
                     "update": true
+                },
+                "default": {
+                    "type": "integer",
+                    "description": "how many entries in the severity-level histogram at the DEFAULT level",
+                    "update": "true"
```

Comment:
DEFAULT is not a level that is used in raising logs.  It is only used in setting the enabled log threshold.  In other words, it should not be counted as it will always be zero.